enemies.js

```diff
@@ -238,4 +238,78 @@ class PirahnaPlant {
             }
         }
     };
-};
\ No newline at end of file
+};
+
+class FireBar {
+    constructor(game, x, y, numOfFires) {
+        Object.assign(this, { game, x, y , numOfFires});
+        let brick = new Brick(this.game, this.x, this.y, 3, "None");
+        this.game.addEntity(brick);
+
+        this.angle = 0;
+        this.fires = [];
+        for (var i = 0; i < this.numOfFires; i++) {
+            var x = i * 25 * Math.sin(this.angle);
+            var y = i * 25 * Math.cos(this.angle);
+            if(i <= 1) {
+                var fire = new FireBar_Fire(this.game, this.x + x + 12, this.y + y + 12, true);
+            } else {
+                var fire = new FireBar_Fire(this.game, this.x + x + 12, this.y + y + 12, false);
+            }
+            this.fires.push(fire);
+            this.game.addEntity(fire);
+        }
+    };
+
+    update() {
+        this.angle += this.game.clockTick;
+        if(this.angle >= 360){
+            this.angle = 0;
+        }
+        console.log(this.numOfFires);
+        for (var i = 0; i < this.numOfFires; i++) {
+            var x = i * 25 * Math.sin(this.angle);
+            var y = i * 25 * Math.cos(this.angle);
+            this.fires[i].x = this.x + x + 12;
+            this.fires[i].y = this.y + y + 12;
+        }
+    };
+
+    drawMinimap(ctx, mmX, mmY) {
```

Comment:
the firebar_fires probably shouldn't show up to avoid cluttering the mini map...but the fire-bar itself might benefit from being added....maybe just draw a 1-pixel orange line that rotates around the block'd origin to show current rotation?

alternatively 2 orange pixels with 1 being the origin and the 2'd used to reflect the outer-most ball's current location?